_posts/2021-03-29-new-release-2021-1.md

```diff
@@ -0,0 +1,11 @@
+---
+title:  "Tableau Connector SDK for 2020.4 Released, latest version of SDK now targets 2021.1"
```

Comment:
Try adjusting the working in the title and abstract to more closely match the [others](https://tableau.github.io/connector-plugin-sdk/news/).  It'll make it more clear we're releasing 2021.1.

Reply:
Updated the title to reflect the wording in title to match other.